wsmaster/che-core-api-agent/src/main/java/org/eclipse/che/api/agent/server/launcher/ProcessIsLaunchedChecker.java

```diff
@@ -39,7 +45,7 @@ public ProcessIsLaunchedChecker(String processNameToWait) {
     @Override
     public boolean isLaunched(Agent agent, InstanceProcess process, Instance machine) throws MachineException {
         Command command = new CommandImpl(format("Wait for %s, try %d", agent.getName(), ++counter),
-                                          format("ps -fC %s 1>/dev/null && echo 0 || echo 1", processNameToWait),
+                                          format(CHECK_COMMAND, processNameToWait),
```

Comment:
multiline command is working ?
I saw many times a very long line with only && in it